src/components/workers-page/workers-tab/workers-tab.component.tsx

```diff
@@ -119,7 +123,12 @@ export default function WorkersTab(): JSX.Element {
                     </span>
                   </TableCell>
                   <TableCell className={classes.tableCell} align="left">
-                    {worker.time}
+                    {ContractTypeHelper.translate((contractType ?? "")[worker.name]) ||
+                      ((!contractType || (contractType && !contractType[worker.name])) &&
+                        "umowa o pracę")}{" "}
+                    {(contractType ?? "")[worker.name] === ContractType.CIVIL_CONTRACT
+                      ? time[worker.name] + " h"
+                      : WorkingTimeHelper.fromHoursToFraction(time[worker.name] * 168, 168)}
```

Comment:
Czy mógłbyś wynieść to do funkcji ( można w tym samym pliku ) i poprzypisywać wszystkie warunki  do zmiennych o sensownej nazwie. 
Bo jeśli w przyszłości zmieni się logika, to będzie zdecydowanie łatwiej jeśli to będzie zapisane w ciut bardziej czytelnej formie niż jest teraz